lib/bill_forward/entities/payment_method.rb

```diff
@@ -1,27 +1,27 @@
 module BillForward
-	class PaymentMethod < MutableEntity
-  		@resource_path = BillForward::ResourcePath.new("payment-methods", "paymentMethod")
+  class PaymentMethod < MutableEntity
```

Comment:
Hm, I think we should prioritize our own preference over Ruby's suggested standard. If we were designing for the "recommended state of the art in Ruby", then we would not have backward compatibility to Ruby 1.8.6 :P
Let's go with tabs.